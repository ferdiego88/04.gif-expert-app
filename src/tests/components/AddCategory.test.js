import AddCategory from "../../components/AddCategory";
import { shallow }from 'enzyme';
describe('Test in AddCategory', () => { 

    const setCategories = ()=> {};
    const wrapper = shallow(<AddCategory setCategories={setCategories} />);

    test('should be show successfully', () => { 
        expect(wrapper).toMatchSnapshot();
     });


     test('should be change inputBox', () => {

        const input = wrapper.find('input');
        const value = "Hola Mundo";
        input.simulate('change',{target: {value}});
        const paragraph = wrapper.find('p');
        
        expect(paragraph.text().trim()).toBe(value);
      })
 })