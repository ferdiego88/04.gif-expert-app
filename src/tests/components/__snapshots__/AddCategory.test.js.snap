// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in AddCategory should be show successfully 1`] = `
<form
  onSubmit={[Function]}
>
  <p>
     
     
  </p>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
